src/components/GettingStarted/DisabledLangPopover.tsx

```diff
@@ -0,0 +1,34 @@
+import { QuestionOutlineIcon } from "@chakra-ui/icons";
+import {
+  IconButton,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverHeader,
+  PopoverBody,
+  PopoverArrow,
+  PopoverCloseButton,
+} from "@chakra-ui/react";
+
+export function DisabledLangPopover() {
+  return (
+    <Popover placement="left-start">
+      <PopoverTrigger>
+        <IconButton
+          aria-label="Disabled language popover"
+          icon={<QuestionOutlineIcon />}
+          variant="ghost"
+        />
+      </PopoverTrigger>
+      <PopoverContent>
+        <PopoverArrow />
+        <PopoverCloseButton />
+        <PopoverHeader>Why are some languages disabled?</PopoverHeader>
+        <PopoverBody>
+          This construct has not yet implemented libraries for all available
```

Comment:
Looking for suggestions on how to better phrase this. I wanted to provide a disclaimer for users to explain why some languages may be disabled so that they would not be confused